README.md

```diff
@@ -1,3 +1,12 @@
+# Deprecated
+
+The `jujutsu` and `jujutsu-lib` crates have been replaced by the `jj-cli`
+and `jj-lib` crates, respectively.
+
+Please upgrade to `jj-cli`/`jj-lib` version 0.8+, or
+specify `jujutsu`/`jujutsu-lib` of version "=0.7.0" if you need to compile an
+obsolete version.
+
```

Comment:
Very optional: I do recommend putting an horizontal rule with "\n-----------\n" here.

Reply:
Done (turns out that `---` gets rendered as a horizontal line).